english/travel-destinations/travel-destinations.md

```diff
@@ -4,4 +4,4 @@ It's Summer! Time to explore these places:
 
 Fall is a great season to take a vacation in any of these spots:
 
-It's frosty winter.. but these places can be excitng:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.